io-engine/src/subsys/config/opts.rs

```diff
@@ -87,15 +91,18 @@ impl From<NvmfTgtConfig> for Box<spdk_nvmf_target_opts> {
         let mut out = Self::default();
         copy_str_with_null(&o.name, &mut out.name);
         out.max_subsystems = o.max_namespaces;
+        out.crdt[0] = o.crdt;
```

Comment:
Should we set o.crdt into all three values i.e. crdt[1] and crdt[2] also? or are we certain that spdk is always using the crdt[0] ?

Reply:
We can consider that in future, but at the moment we don't have a way to use CRDT 2,3 because SPDK always responds with CRDT #1: https://github.com/openebs/spdk/blob/v23.01.x-mayastor/lib/nvmf/ctrlr.c#L4343